grid.cpp

```diff
@@ -342,6 +372,17 @@ void initializeGrids(
    momentsGrid.updateGhostCells();
    momentsDt2Grid.updateGhostCells();
    phiprof::stop("Finish fsgrid setup");
+
+   // Set this so CFL doesn't break
+   if(P::adaptRefinement) {
+      // Half-step acceleration
+      if( P::propagateVlasovAcceleration ) {
+         calculateAcceleration(mpiGrid, -0.5*P::dt + 0.5*P::bailout_min_dt);
+      } else {
+         calculateAcceleration(mpiGrid, 0.0);
+      }
+      P::dt = P::bailout_min_dt;
+   }
```

Comment:
The `else` block is just what's used in `vlasiator.cpp`, so I copied it here as well.

Reply:
Yes, I saw that, and there we need it, here I am less sure. But it shouldn't hurt.

Reply:
with a dt of 0, calculateacceleration just a) applies sparsity and b) re-calculates moments. I don't think either of those is needed here.

It would be better IMO to actually evaluate max CFLs and set that instead of the bailout dt, even though it is used for only one step.

Reply:
That's still a valid point I suppose?